app/src/main/java/com/simplecity/amp_library/playback/MusicService.java

```diff
@@ -639,7 +639,7 @@ public void onCreate() {
                             playerHandler.sendEmptyMessage(MusicService.PlayerHandler.FADE_DOWN_STOP);
                         }
                     }
-                }, throwable -> LogUtils.logException(TAG, "rror consuming SleepTimer observable", throwable)));
+                }, throwable -> LogUtils.logException(TAG, "Error consuming SleepTimer observable", throwable)));
```

Comment:
This isn't related to the current PR, should be made under a separate commit/PR. (You can leave it as it is for this PR, just for future reference)